src/swap/SwapScreen.test.tsx

```diff
@@ -1180,12 +1177,47 @@ describe('SwapScreen', () => {
     expect(within(tokenBottomSheet).queryByText('Test Token')).toBeFalsy()
   })
 
-  it('should disable buy amount input when swap buy amount experiment is set is false', () => {
+  it('should not show input sections if tokens are not selected', () => {
     jest.mocked(getExperimentParams).mockReturnValue({
       swapBuyAmountEnabled: false,
     })
     const { swapFromContainer, swapToContainer } = renderScreen({})
 
+    expect(within(swapFromContainer).queryByTestId('SwapAmountInput/Input')).toBeFalsy()
+    expect(within(swapToContainer).queryByTestId('SwapAmountInput/Input')).toBeFalsy()
+  })
+
+  it('should be able to switch tokens by pressing arrow button', async () => {
+    jest.mocked(getExperimentParams).mockReturnValue({
+      swapBuyAmountEnabled: false,
+    })
+    const { swapFromContainer, swapToContainer, swapScreen, getByTestId } = renderScreen({})
+
+    selectSwapTokens('CELO', 'cUSD', swapScreen)
+
+    expect(within(swapFromContainer).getByTestId('SwapAmountInput/Input')).toBeTruthy()
+    expect(within(swapFromContainer).getByText('CELO')).toBeTruthy()
+
+    expect(within(swapToContainer).getByTestId('SwapAmountInput/Input')).toBeTruthy()
+    expect(within(swapToContainer).getByText('cUSD')).toBeTruthy()
```

Comment:
nit: `selectSwapTokens` [already includes](https://github.com/valora-inc/wallet/blob/5946f1950f97028282765c31c2319c4cdfd6eabf/src/swap/SwapScreen.test.tsx#L251) these assertions so you don't need to do it again here :)